### ISSUE TITLE
90 min PPP 2.22

### ISSUE THREAD

**Issue by anish-kum** [anish-kum/PPP-Spark-2025-2026#28]
90 min PPP

**Comment by anish-kum** [comment]
<img width="336" height="260" alt="Image" src="https://github.com/user-attachments/assets/ad6e63b6-481f-4155-8578-5ea0e18b32fd" />

<img width="338" height="226" alt="Image" src="https://github.com/user-attachments/assets/4967751f-396d-42e1-8f7c-8e8057cabafc" />

What I learned:

- Style Transfer Overview: Discusses the concept of style transfer, where the style of one image is applied to the content of another. This is important because it showcases the creative potential of neural networks in generating visually appealing images.

- Content and Style Representations: Explains how content and style are represented using convolutional neural networks (CNNs). This is important because understanding these representations is crucial for effectively combining content and style in images.

- Loss Functions in Style Transfer: Introduces the content loss and style loss functions used to guide the optimization process. This is important because these loss functions determine how well the generated image matches the desired content and style.

- Optimization Process: Describes the iterative process of adjusting the generated image to minimize the defined loss functions. This is important because it highlights the practical application in achieving the desired artistic effect in style transfer.

2 Questions I still had:

- What happens to the final image if the style loss is much bigger than the content loss (the "loss" in style loss and content loss means the weightings of these values when calculating the total loss for training the model; this is oversimplified)?

- Why do deeper CNN layers capture content better while earlier layers capture style?
